lib/phoenix/digester.ex

```diff
@@ -118,8 +118,11 @@ defmodule Phoenix.Digester do
   defp manifest_join(path, filename), do: Path.join(path, filename)
 
   defp compiled_file?(file_path) do
+    compressors = Application.fetch_env!(:phoenix, :static_compressors)
+    compressed_extensions = Enum.map(compressors, &(&1.file_extension))
```

Comment:
It may be worth calling it `file_extensions` and using `flat_map` in case one format can work with mutliple extensions.

Reply:
And then we pick the first one when writing the file to disk.

Reply:
done!